src/bundle/Resources/config/services/components.yaml

```diff
@@ -16,6 +16,13 @@ services:
         arguments:
             $parameters: []
 
+    EzSystems\EzPlatformAdminUi\Component\TabsComponent:
+        abstract: true
+        lazy: true
+        arguments:
+            $template: '@@ezdesign/ui/tab/default.html.twig'
+            $parameters: []
```

Comment:
```suggestion
```

`[]` is already a default value for `$parameters` parameter in `EzSystems\EzPlatformAdminUi\Component\TabsComponent::__construct`